helm/korifi/controllers/crds/korifi.cloudfoundry.org_appworkloads.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.16.4
+    controller-gen.kubebuilder.io/version: v0.16.3
```

Comment:
So your local ./controllers/bin/controller-gen is not using 0.16.4?

Reply:
Nice catch, @gogolok - I totally overlooked that this is not a bump, but de-bump. Will fix that